lib/tsd-file-api-client/build.gradle.kts

```
@@ -0,0 +1,39 @@
+plugins {
+    id("java")
+}
```

```suggestion
    id("formatting-conventions")
}
```
Make sure that formatting styles are adhered to.